ParameterInput.js

```diff
@@ -65,6 +72,10 @@ function parameterInputFactory(label, inputType, initialValue, domain, onValueCh
     return new TextParameterInput(label, initialValue, domain, onValueChanged, containerId);
   }
 
+  if (inputType === inputTypes.click) {
+    return new ClickParameterInput(onValueChanged, containerId);
+  }
+
   // TODO, implement class ClickParameterInput and delete test_graph_plot.js and test_contour_plot.js
```

Comment:
```suggestion

```
We can probably delete test_graph_plot.js and test_contour_plot.js and remove this comment now 🎉